src/index.ts

```diff
@@ -1,11 +1,13 @@
 import OctokitApp from '@octokit/app'
+import Octokit from '@octokit/rest'
```

Comment:
I think that maybe we should export the bare `Octokit` class here, so that people who want to pass their customized Octokit with custom defaults and plugins don’t need to install `@octokit/rest` as dependency and don’t end up passing an Octokit version which is incompatible with what probot does

Reply:
They don’t need to install it anyway. It is dependency of Probot. In my project I’ve listed only Probot. Other libs are part of it. 

But if you’d like to do export it, no problem.

Reply:
Done 😉